### ISSUE TITLE
cannot gdownload any images

### ISSUE THREAD

**Issue by qianwangn** [joelthchao/tensorflow-finetune-flickr-style#3]
Downloading 80000 images with 3 workers...
Writing train/val for 0 successfully downloaded images.

**Comment by joelthchao** [comment]
Original image source (e.g. http://farm3.staticflickr.com/2887/12524447173) is somehow dead, which affects Caffe's implement as well. Plan to find a available source to replace it.

**Comment by micklexqg** [comment]
the same error: Downloading 80000 images with 7 workers...
Writing train/val for 0 successfully downloaded images.
Dear author, Have you solved it?

**Comment by joelthchao** [comment]
No, the original dataset seems to be dead forever. Need to switch to other dataset.

**Comment by micklexqg** [comment]
thanks, would you like to paste a screenshot picture of the txt content? I want to generate them with another dataset.

**Comment by joelthchao** [comment]
First column is path to image file and second column is class image belongs to (0-index)
```
path_to_img_1 0
path_to_img_2 1
path_to_img_3 0
path_to_img_4 2
```

**Comment by micklexqg** [comment]
just like this?
![image](https://user-images.githubusercontent.com/13776012/29484413-c1414a72-84f0-11e7-9350-ff2b92c17d39.png)

**Comment by joelthchao** [comment]
format is correct, but how come you have so many classes? 30131?

**Comment by micklexqg** [comment]
@joelthchao , the given label ID is not from 0 to 1000. maybe it's pre-defined.

**Comment by joelthchao** [comment]
You will need to map it into 0 to num_class - 1, or the code won't work.

**Comment by micklexqg** [comment]
yes, you are right,it doesn't work. Redefine the label? are there any methods if we don't change the original label?

**Comment by joelthchao** [comment]
Yes, you need to re-define the label. Or modify `dataset.py` to achieve it.

**Comment by joelthchao** [comment]
@johnlanbor @micklexqg Download image data issue has been fixed in the latest commit. Please read the README to check the new usage.
